css-layout-classes/styles.css

```diff
@@ -0,0 +1,45 @@
+.container {
+  margin: auto;
+  min-width: 0;
+}
+
+h1 {
+  font-size: 20px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.margin75 {
+  margin: 0 75px;
+}
+
+h2 {
+  font-size: large;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+p {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 12px;
+}
+
+.row {
+  display: flex;
+  min-width: 0;
+  align-items: center;
+}
```

Comment:
Don't pollute your layout classes. The `row` class should only have `display: flex` applied to it. You should make a separate rule for this additional styling.